src/test/ui-fulldeps/lint_tool_test.stderr

```diff
@@ -1,8 +1,62 @@
-warning: item is named 'lintme'
-  --> $DIR/lint_tool_test.rs:19:1
+warning: lint name `test_lint` is deprecated and may not have an effect in the future. Also `cfg_attr(cargo-clippy)` won't be necessary anymore
+  --> $DIR/lint_tool_test.rs:18:23
    |
-LL | fn lintme() { } //~ WARNING item is named 'lintme'
+LL | #![cfg_attr(foo, warn(test_lint))]
+   |                       ^^^^^^^^^ help: change it to: `clippy::test_lint`
+   |
+   = note: #[warn(renamed_and_removed_lints)] on by default
```

Comment:
The reason that this warning is emitted twice is the note `#[warn(renamed_and_removed_lints)] on by default`, which only appears on the first occurrence of lint warnings. That doesn't effect the `UNKNOWN_LINTS` lint though.. :thinking: